apps/site/assets/ts/stop/__tests__/DepartureListTest.tsx

```diff
@@ -70,11 +70,15 @@ describe("DepartureList", () => {
       />
     );
 
-    expect(screen.getByText(schedules[2].time.toString())).toBeDefined();
+    expect(container.querySelector("time")).toBeDefined();
+    expect(container.querySelectorAll("time")[2]).toHaveAttribute(
+      "datetime",
+      schedules[2].time.toISOString()
+    );
```

Comment:
Is there a specific reason that `container` is used here to parse the dom instead of the `screen.get/queryBy()` functions?  I think the test would be better if it queried what the display string should be.

Reply:
oh!!! yeah I first tried get/queryByRole and it didn't work; I don't think the library supports the `<time>` role yet. and in some cases, I didn't want to query on specific text displayed, because the text might be displayed differently based on many things, whereas the `datetime` attribute will be consistently formatted... so to me that ruled out the get/queryByText family of functions. should I add a `data-testid` for this case maybe?

Reply:
I just pushed a change taking a slightly different approach with this test to avoid needing to use `container`.